polaris-react/src/components/Frame/Frame.scss

```
@@ -5,6 +5,7 @@
   --pc-frame-button-size: var(--p-space-8);
   width: 100%;
   min-height: 100vh;
+  min-height: 100svh; // For mobile browsers, fill the screen taking into account dynamic browser chrome
```

Should we use the dynamic viewport unit so the change in viewport height is respected when expanding and retracting?

```suggestion
  min-height: 100dvh;
```

---

(Ah ignore me, dvh is too unstable)